app/assets/stylesheets/application.scss

```diff
@@ -573,3 +573,7 @@ table {
 .masked-input {
   font-family: 'AllBullets';
 }
+
+.code-in-table {
```

Comment:
If you're intending to change the colour then I'd suggest either not doing it, using `<code>` tags around individual items in the list to better separate them, or extend the `code` CSS within the table to change the color/background properties.